end_to_end_tests/golden-record-custom/custom_e2e/client.py

```diff
@@ -30,9 +30,15 @@ def with_cookies(self, cookies: Dict[str, str]) -> "Client":
     def get_timeout(self) -> float:
         return self.timeout
 
-    def with_timeout(self, timeout: float) -> "Client":
-        """ Get a new client matching this one with a new timeout (in seconds) """
-        return attr.evolve(self, timeout=timeout)
+    def with_timeout(self, timeout: float, override_lower_default: bool = True) -> "Client":
```

Comment:
nit: `override_lower_default` is a confusing name to me; I'm reading it as though your supplied value should be overriding a lower one -- increasing the timeout, which isn't the purpose of this change.

Maybe `increase_only: bool = False`?

Reply:
Agree here that the name is a bit confusing